mappings/net/minecraft/network/listener/ClientPlayPacketListener.mapping

```diff
@@ -36,7 +36,7 @@ CLASS net/minecraft/class_2602 net/minecraft/network/listener/ClientPlayPacketLi
 		ARG 1 packet
 	METHOD method_11094 onBlockEntityUpdate (Lnet/minecraft/class_2622;)V
 		ARG 1 packet
-	METHOD method_11095 onEntitiesDestroy (Lnet/minecraft/class_2716;)V
+	METHOD method_11095 onEntityDestroy (Lnet/minecraft/class_2716;)V
```

Comment:
This packet was destroying multiple entities I recall until mojang did a bugfix.

Reply:
Yep, can confirm that (21w17a, #2355)